test/jdk/ProblemList.txt

```diff
@@ -256,8 +256,7 @@ sun/java2d/SunGraphics2D/SimplePrimQuality.java 6992007 generic-all
 sun/java2d/SunGraphics2D/SourceClippingBlitTest/SourceClippingBlitTest.java 8196185 generic-all
 
 sun/java2d/X11SurfaceData/SharedMemoryPixmapsTest/SharedMemoryPixmapsTest.sh 7184899,8221451 linux-all,macosx-aarch64
-java/awt/FullScreen/DisplayChangeVITest/DisplayChangeVITest.java 8169469,8273617 windows-all,macosx-aarch64
-java/awt/FullScreen/UninitializedDisplayModeChangeTest/UninitializedDisplayModeChangeTest.java 8273617 macosx-all
+java/awt/FullScreen/DisplayChangeVITest/DisplayChangeVITest.java 8169469 windows-all,macosx-aarch64
```

Comment:
So does this test really still fail on macos ARM ?
The only remaining bug ID is entirely about windows.
Seems you should remove the arm entry and re-test and it if does fail, then maybe the test isn't right yet.

Reply:
Also, `DisplayChangeVITest`  doesn't fail locally in my macOS ARM 15.7 so it is another CI issue..probably timeout increase will help but it needs to be done w.r.t 8169469 or another JBS but probably not in this PR which is about `UninitializedDisplayModeChangeTest`

Reply:
DisplayChangeVITest mac ARM PL entry updated with new id 8368736